vendor/composer/autoload_classmap.php

```diff
@@ -6,4 +6,7 @@
 $baseDir = dirname($vendorDir);
 
 return array(
+    'Config_File' => $vendorDir . '/smarty/smarty/libs/Config_File.class.php',
+    'Smarty' => $vendorDir . '/smarty/smarty/libs/Smarty.class.php',
+    'Smarty_Compiler' => $vendorDir . '/smarty/smarty/libs/Smarty_Compiler.class.php',
```

Comment:
Yes, when you run `composer install` it also sets the autoload. When you bring in new classes you can also reset the autoload by doing a `composer dump-autoload`
[https://getcomposer.org/doc/03-cli.md#dump-autoload](https://getcomposer.org/doc/03-cli.md#dump-autoload)